provider.swagger.yml

```diff
@@ -146,16 +146,17 @@ definitions:
         description: the full url to access the consultation instance
       name:
         type: string
-        description: name of the instance (as it was provided by the client, even if it gets altered later in the instance lifecycle)
+        description: name of the instance
       adminName:
         type: string
-        description: name of the instance admin user (as it was provided by the client, even if it gets altered later in the instance lifecycle)
+        description: name of the instance admin user
       adminEmail:
         type: string
         format: email
-        description: email of the instance admin user (as it was provided by the client, even if it gets altered later in the instance lifecycle)
+        description: email of the instance admin user
```

Comment:
Repoussé, reformulé ainsi: (may be in sync with the running instance, or as given in the request)